refinery/tool_manager/utils.py

```diff
@@ -245,3 +248,52 @@ def validate_workflow_step_annotation(workflow_step_dictionary):
         raise RuntimeError(
             "{}{}".format(ANNOTATION_ERROR_MESSAGE, e)
         )
+
+
+def get_workflow_list():
+    """
+    Generate a list of available workflows from all currently available
+    WorkflowEngines
+    :return: list of workflow dicts
+    """
+    workflow_list = []
+    workflow_engines = WorkflowEngine.objects.all()
+
+    sys.stdout.write(
```

Comment:
We should probably make this utility function re-useable in other parts of the application besides management commands. If so, it should not be writing anything to stdout.